examples/frame/src/in_frame/index.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright 2025 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+
+import { type InitData, root, useInitData } from "@lynx-js/react";
+import type { GlobalProps } from "@lynx-js/types";
+
+export function App() {
+  let btsInitData: InitData = useInitData();
+  let globalProps: GlobalProps = lynx.__globalProps;
+
+  return (
+    <view>
+      <text style={{ fontSize: "30px", color: "orange" }}>I'm in frame!</text>
+      <view style={{ border: "3px solid blue" }}>
+        <text>{btsInitData?.id}</text>
+        <text>{btsInitData?.name}</text>
+        <text>{btsInitData?.showDetail ? "true" : "false"}</text>
+        <text>{globalProps?.message}</text>
+        <text>{globalProps?.status}</text>
+        <text>{globalProps?.loaded ? "true" : "false"}</text>
```

Comment:
The optional chaining (`?.`) here does not match the type definition of `InitData` and `GlobalProps`. Should we just remove it?


```suggestion
        <text>{btsInitData.id}</text>
        <text>{btsInitData.name}</text>
        <text>{btsInitData.showDetail ? "true" : "false"}</text>
        <text>{globalProps.message}</text>
        <text>{globalProps.status}</text>
        <text>{globalProps.loaded ? "true" : "false"}</text>
```